src/app/core-components/popup-host/popup-host.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="popup-wrapper" [ngClass]="{displayPopup: isEmptyTemplate}">
```

Comment:
Тут просто подсвечу подход с binding классов, на мой взгляд удобный и более читаемый подход
```suggestion
<div class="popup-wrapper" [class.displayPopup]="isEmptyTemplate">
```